modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/util/test/CalendarUtilTest.java

```diff
@@ -204,6 +207,49 @@ null, createThemeDisplay(), calendarBookingInstance,
 		assertRepeatsForever(recurrence);
 	}
 
+	@Test
+	public void testToCalendarBookingJSONObjectVulnerabilities()
+		throws Exception {
+
+		ServiceContext serviceContext = createServiceContext();
+
+		HashMap<Locale, String> injectedContent = HashMapBuilder.put(
+			LocaleUtil.getDefault(),
+			"lp'\"></option><img src=x onerror=alert(document.location)>"
```

Comment:
Hi @rijema Brian asked me to sort the attributes of the tag, so I'll ask you to sort here as well, for example:

The string should be:

`"lp'\"></option><img onerror=alert(document.location) src=x >"`

note the `onerror` attribute should comes before `src`.

Thank you!

Reply:
Done. Thanks! Could you please check?